scripts/abilities.gd

```diff
@@ -672,6 +672,94 @@ var abilitydict = {
 		aitargetcond = 'any',
 		aipriority = 2,
 	},
+####---Added by Expansion---#### CyrusLance's combat spells
+	arcanebolt = {
+		name = 'Arcane Bolt',
+		code = 'arcanebolt',
+		iconnorm = load("res://files/aric_expansion_images/skill_icons/Arcanebolt.png"),
+		iconpressed = load("res://files/aric_expansion_images/skill_icons/Arcanebolt2.png"),
+		icondisabled =load("res://files/aric_expansion_images/skill_icons/Arcanebolt3.png"),
+		learnable = true,
+		description = "Quick cast spell deals damage to a selected target, scales with Magic Affinity. ",
+		usetext = '$name [color=aqua]uses Arcane Bolt[/color] at $targetname. ',
+		target = 'one',
+		targetgroup = 'enemy',
+		effect = null,
+		can_miss = false,
+		learncost = 30,
+		power = 1,
+		cooldown = 0,
+		type = 'spell',
+		price = 50,
+		costenergy = 0,
+		costmana = 0,
+		costother = '',
+		attributes = ['damage'],
+		reqs = {'level' : 1, 'smaf' : 1},
+		aipatterns = ['attack'],
+		aitargets = '1enemy',
+		aiselfcond = 'any',
+		aitargetcond = 'any',
+		aipriority = 6,
+	},
+	astralpoison = {
+		name = 'Astral Poison',
+		code = 'astralpoison',
+		iconnorm = load("res://files/aric_expansion_images/skill_icons/Astralpoison.png"),
+		iconpressed = load("res://files/aric_expansion_images/skill_icons/Astralpoison2.png"),
+		icondisabled =load("res://files/aric_expansion_images/skill_icons/Astralpoison3.png"),
+		learnable = true,
+		description = "Envelop enemies with corrupting mana inflicting curse upon them, scales with Magic Affinity. ",
+		usetext = '$name [color=aqua]uses Astral Poison[/color] shrouding the enemies on battlefield. ',
+		target = 'all',
+		targetgroup = 'enemy',
+		effect = 'curseeffectatk',
+		can_miss = false,
+		learncost = 50,
+		power = 0.9,
+		cooldown = 3,
+		type = 'spell',
+		price = 150,
+		costenergy = 0,
+		costmana = 20,
+		costother = '',
+		attributes = ['damage','debuff'],
+		reqs = {'level' : 4, 'smaf' : 4},
+		aipatterns = ['attack'],
+		aitargets = '1enemy',
```

Comment:
```suggestion
		aitargets = 'enemies',
```
Here and in Nova you probably want multiple target for ai, I think. Not that they will use these spells, but still.

Reply:
Nice catch, overlook that part.